### DISCUSSION TITLE
Possible to use this library without mordant?

### DISCUSSION THREAD

**Discussion by Kshitij09-sc** [ajalt/clikt#537] [Q&A]
I'm aiming for smallest binary size for a CLI application and while using this library on JVM noticed majority of size contribution was coming from `.so` files bundled with mordant. Is it possible to publish a lightweight artifact of clikt with bare minimum argument parsing?

**Comment by ajalt** [comment]
This is already implemented in master. You can use it now with the [latest snapshot][1] e.g. `implementation("com.github.ajalt.clikt:clikt-core:4.4.0.37-SNAPSHOT")`

[1]: https://github.com/ajalt/clikt?tab=readme-ov-file#snapshots
